radix-engine/tests/preview.rs

```diff
@@ -0,0 +1,97 @@
+#[rustfmt::skip]
+pub mod test_runner;
+
+use crate::test_runner::TestRunner;
+use radix_engine::engine::{RuntimeError, TransactionCostCounterConfig, TransactionExecutorConfig};
+use radix_engine::fee::{CostUnitCounterError, DEFAULT_MAX_TRANSACTION_COST};
+use scrypto::core::Network;
+use transaction::builder::ManifestBuilder;
+use transaction::builder::TransactionBuilder;
+use transaction::model::*;
+use transaction::signing::EcdsaPrivateKey;
+use transaction::validation::{TestEpochManager, TestIntentHashManager, TransactionValidator};
+
+#[test]
+fn test_transaction_preview_cost_estimate() {
```

Comment:
Can we split test to be more specific about what is being tested and use Arrange/Act/Assert pattern?

Reply:
I've reorganized it a bit. Is: Arrange, (Act & Assert #1), (Act & Assert #2) acceptable? I think it's more readable that way.

Reply:
on second thought, that first assert probably isn't needed anymore, since the preview config doesn't even specify the system loan amount